repos/archive/rust-lang/gll.toml

```diff
@@ -0,0 +1,6 @@
+org = "rust-lang"
+name = "gll"
+description = " GLL parsing framework."
```

Comment:
Should we put something on the README and/or description explaining any detail about why this is archived? In other organizations I'm a part of, we make a habit of putting a prominent message in the README explaining where those looking at the repo should direct their attention.

Reply:
Sure! I posted https://github.com/rust-lang/gll/pull/145 and https://github.com/rust-lang/wg-grammar/pull/70. However, I'm not sure how active @eddyb is to be able to see this.

Reply:
> However, I'm not sure how active @eddyb is to be able to see this.

My notifications are a disaster because I didn't get myself out of @rust-lang/compiler the moment I stopped being able to pay attention to them, so while I keep every GitHub email for posterity, I have no easy way to be notified when people want me to look at anything under `rust-lang/`.
You pretty much have to DM me on Zulip or Discord for the time being, sorry!
<sub>(I've been trying to fix it but life keeps handing me lemons - also, theoretically the `@eddyb` on GitHub may have been enough to bypass that but maybe I need to do more email automation because I don't remember the GH mobile app catching it, and it's also terrible in other ways)</sub>

---

Also, IMO `gll` does not belong in `rust-lang/`, it's a toy parsing library developed originally under the `@LykenSol` org and it got moved because wg-grammar wanted some level of "vendoring". Completely unnecessary and it would be best to move it back if that's an option. (see also https://github.com/rust-lang/gll/pull/145#issuecomment-2039830588)